Update readme.md

muydanny opened the pull request:
# Backend Curriculum PR Template

### Description

This change directs people to the new curriculum site and github. It also adds to check out Turing.edu.

### Why are we making this update?

New site
  
### Type of update

- [ ] Minor update/fix -- no review requested
- [ ] Moderate update -- review from Mod Team requested
- [ ] Major update -- review from Backend Team requested

### How will we measure the success of this change? 

digital high fives

### What questions do you have/what do you want feedback on? (optional)

*List any specific questions and feedback requests*

AliSchlereth commented on module2/misc/blogger.md:
````diff
@@ -1056,6 +1056,9 @@ end
 This runs the `set_article` method before `destroy` and `edit`, as specified by the `only:` key. With this, `@article` is available within the scopes of th `destroy` and `edit` actions. We can now remove the lines in `destroy` and `edit` that set `@article`, leaving something like this:
 
 ```ruby
+def show
````
Thanks @Maxscores. Could you add show to the above paragraph about removing the redundant lines. Also while you're at it, it looks like there is a typo from a previous version where 'th' should be 'the'.

Maxscores replied:
@AliSchlereth Thanks for the comment. I've committed those changes to the PR.

AliSchlereth commented on module2/misc/blogger.md:
````diff
@@ -1053,9 +1053,12 @@ class ArticlesController < ApplicationController
 ```
 
-This runs the `set_article` method before `destroy` and `edit`, as specified by the `only:` key. With this, `@article` is available within the scopes of th `destroy` and `edit` actions. We can now remove the lines in `destroy` and `edit` that set `@article`, leaving something like this:
+This runs the `set_article` method before `show`, `destroy` and `edit`, as specified by the `only:` key. With this, `@article` is available within the scopes of the `show`, `destroy` and `edit` actions. We can now remove the lines in `show`, `destroy` and `edit` that set `@article`, leaving something like this:
````
👍

muydanny committed:
Update readme.md

This change directs people to the new curriculum site and github. It also adds to check out Turing.edu.
